lib/commands/project/project-command-base.ts

```diff
@@ -0,0 +1,12 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+export class ProjectCommandBase {
```

Comment:
Maybe we can use `canExectute`, but I need to copy paste this check in `create.ts` and in `create-project.ts`.  I need it to be only on one place.